hoomd/logging.py

```diff
@@ -21,35 +32,35 @@ class LoggerCategories(Flag):
     categories to use for limiting what data is logged, user specified logged
     quantities, and custom actions (`hoomd.custom.Action`).
 
-    Flags:
-        scalar: `float` or `int` objects (i.e. numbers)
+    Attributes:
+        scalar: 1 `float` or `int` objects (i.e. numbers)
```

Comment:
Putting the numbers right before the description is a bit confusing and hard to read. How about changing it like so?
```suggestion
        scalar: `float` or `int` objects (i.e. numbers) (enum value: 1)
```